ios/packages/test-utils/Sources/ui-test/AssetAndPluginCollection.swift

```diff
@@ -0,0 +1,274 @@
+//
+//  AssetCollectionViewController.swift
+//  PlayerUI
+//
+//  Created by Harris Borawski on 2/18/21.
+//
+
+import Foundation
+import SwiftUI
+import Combine
+
+/**
+ A SwiftUI View to load flows for ease of UI testing
+ */
+public struct AssetAndPluginCollection: View {
+    let sections: [FlowLoader.FlowSection]
+    let padding: CGFloat
+
+    /**
+     Initializes and loads flows
+     - parameters:
+        - plugins: Plugins to add to Player instance that is created
+        - sections: The `[FlowSection]` to display
+        - completion: A handler for when a flow reaches an end state
+     */
+    public init(
+        sections: [FlowLoader.FlowSection],
+        padding: CGFloat = 16
+    ) {
+        self.padding = padding
+        self.sections = sections
+    }
+
+    enum HeaderSelection: String, CaseIterable {
+        case flows = "Flows"
+        case player = "Player"
+    }
+
+    @State var segmentationSelection: HeaderSelection = .flows
+
+    @State var pubsubEventPublished = false
+    @State var pubsubEventName = ""
+    @State var pubsubEventMessage = ""
+
+    @State var beaconsRecieved = false
+    @State var beaconsInfo = ""
+
+    @State var doneFlow = false
+
+    public var body: some View {
+        VStack {
+            Picker("", selection: $segmentationSelection) {
+                ForEach(HeaderSelection.allCases, id: \.self) { option in
+                    Text(option.rawValue)
+                }
+            }.pickerStyle(SegmentedPickerStyle())
+
+            Spacer()
+
+            if segmentationSelection == .flows {
+                flowsListSection
+            } else {
+                playerListSection
+            }
+        }
+    }
+
+    var flowsListSection: some View {
+        return List {
+            ForEach(sections, id: \.title) { section in
+                Section {
+                    ForEach(section.flows, id: \.name) { flow in
+                        NavigationLink(flow.name) {
+                            PlayerLoader(flow: flow.flow)
+                                .navigationBarTitle(Text(flow.name))
+                        }
+                        .accessibility(identifier: "\(section.title) \(flow.name)")
+                    }
+                } header: {
+                    Text(section.title)
+                }
+            }
+        }
+        .accessibility(identifier: "AssetAndPluginCollection")
+        .navigationBarTitle(Text("Flows"))
+    }
+
+    var playerListSection: some View {
+        List {
```

Comment:
These utilities are published for consuming users to use, this would mean that all users using the utility will now have a section for these hardcoded managedplayer flows if they were to use this.

This will need to be refactored so that we can show those flows in our demo app, using the same tools we publish for consuming users. But we do not want to include our plugin examples in other users demo/example apps.

In my mind, we would not have changed `AssetCollectionView`, we would provide an additional utility to setup more specific flows, and have it use `AssetCollectionView` to display the asset flows as it does today.